README.md

```diff
@@ -0,0 +1,6 @@
+Hello World
```

Comment:
Requires Changes ♻️
- Required linter check for error for all projects building.

Reply:
- [ ] It is always nice to make your comments suggestive or having polite phrases, instead of commanding sentences. In addition, you should always write comments highlighting the line or block of code related to your comments. I don't think linter has anything to do with the README file.

Reply:
Linter should be used to correct errors in the code and make the code more efficient with the browser.

Reply:
Good!